src/backend/web/routes/feeds.js

```diff
@@ -51,4 +52,26 @@ feeds.get('/:id', async (req, res) => {
   }
 });
 
+feeds.post('/', protect, async (req, res) => {
+  const feedData = req.body;
+  if (!feedData.url && !feedData.author) {
+    res.status(400).json({ message: `URL and Author must be submitted` });
+  }
+  if (await Feed.byUrl(feedData.url)) {
+    res.status(409).json({ message: `Feed for url ${feedData.url} already exists.` });
+    return;
+  }
+  try {
+    const feed = await Feed.create(feedData);
```

Comment:
Since what you get back from `Feed.create` is an `id` for your feed, let's call it that instead of `feed`: `const id = ...`